api/management/api.go

```diff
@@ -611,6 +611,8 @@ func ConvertRealmAdminConfigurationFromDBStruct(conf configuration.RealmAdminCon
 	var checks = conf.AvailableChecks
 	if checks == nil {
 		checks = make(map[string]bool)
+		checks[configuration.CheckKeyIDNow] = false
```

Comment:
You also can write it with a single instruction:
checks = map[string]bool{configuration.CheckKeyIDNow: false, configuration.CheckKeyPhysical: false}